tests/data/metadata/discovery/surface-weather-observations.yml

```diff
@@ -10,7 +10,7 @@ mcf:
     version: 1.0
 
 metadata:
-    identifier: observationsSurfaceLand.mw.FWCL.landFixed.hourly
+    identifier: observations-surface-land.mw.FWCL.landFixed
```

Comment:
This should stay as `observationsSurfaceLand.mw.FWCL.landFixed.hourly`; given `hourly` is part of the dataset name (vs. `observations-surface-land.mw.FWCL.landFixed` which is the topic hierarchy of the given dataset name).